resources/views/Checkout/CheckoutView.twig

```diff
@@ -125,23 +125,31 @@
                                     <h3>{{ trans("Ceres::Template.checkoutShippingProfile") }}</h3>
                                 </header>
 
+                                {% set overridePaymentList = LayoutContainer.show("Ceres::Checkout.PaymentList") %}
+                                {% if  overridePaymentList|trim  is empty %}
+                                    {% set isOverwritten = false %}
+                                {% else %}
+                                    {% set isOverwritten = true %}
+                                {% endif %}
+
                                 {{ LayoutContainer.show("Ceres::Checkout.BeforeShippingProfileList") }}
                                 {% if overrideShippingProfiles|trim is empty %}
                                     <div class="shipping-method-select">
-                                        <shipping-profile-select></shipping-profile-select>
+                                        <shipping-profile-select
+                                            :payment-container-is-overwritten="{{ isOverwritten | json_encode }}"
+                                        ></shipping-profile-select>
                                     </div>
                                 {% else %}
                                     {{ overrideShippingProfiles }}
                                 {% endif %}
                                 {{ LayoutContainer.show("Ceres::Checkout.AfterShippingProfileList") }}
 
-                                {% set overridePaymentList = LayoutContainer.show("Ceres::Checkout.PaymentList") %}
                                 <header class="mb-3">
                                     <h3>{{ trans("Ceres::Template.checkoutPaymentMethod") }}</h3>
                                 </header>
 
                                 {{ LayoutContainer.show("Ceres::Checkout.BeforePaymentList") }}
-                                {% if overridePaymentList|trim is empty %}
+                                {% if isOverwritten is same as(false)  %}
```

Comment:
geht hier nicht einfach `{% if not isOverwritten %}` ?

Reply:
teste ich